libs/platform/experience/src/models/composition.model.ts

```diff
@@ -4,6 +4,12 @@ import { LayoutAttributes } from '../../layout/src';
 
 export interface CompositionProperties {
   rules?: StyleRuleSet[];
+  visibility?: ComponentVisibility;
```

Comment:
Is this model flexible enough to support:
- multiple conditional rules, e.g.: `hide when x,  show when y`; `hide when x AND y`; `hide when x OR y`.
- parameterized rules, e.g.: `hide when route = checkout` (semantic link ref); `show when cart entries > 5`; `hide when screen size = sm`

I'm not asking to build anything link this, just to ensure that we can extend the data model going forward without breaking changes.